roles/openshift_control_plane/defaults/main.yml

```diff
@@ -8,7 +8,7 @@ r_openshift_master_firewall_enabled: "{{ os_firewall_enabled | default(True) }}"
 r_openshift_master_use_firewalld: "{{ os_firewall_use_firewalld | default(False) }}"
 
 l_openshift_images_dict:
-  origin: 'docker.io/openshift/origin-${component}:${version}'
+  origin: 'openshift/origin-${component}:${version}'
```

Comment:
I believe we want the docker.io portion here still.

Reply:
This breaks CI, as `docker_image` would look for an image called `docker.io/openshift/origin-node:deadbeef` and try to pull it instead.

This doesn't seem to work without docker.io, so I'm still experimenting with this